File: utilsLibrary/src/main/java/com/zeropercenthappy/utilslibrary/LogUtils.java
package com.zeropercenthappy.utilslibrary;

import android.util.Log;

/**
 * Created by Administrator on 2017/4/1.
 */

public class LogUtils {
    private static String LogTag = "debug";

    private static boolean enable = false;

    public static boolean isEnable() {
        return enable;
    }

    public static void setEnable(boolean enable) {
        LogUtils.enable = enable;
    }

    public static String getLogTag() {
        return LogTag;
    }

    public static void setLogTag(String logTag) {
        LogTag = logTag;
    }

    public static void i(String content) {
        if (enable) {
            Log.i(LogTag, content);
        }
    }

    public static void i(String tag, String content) {
        if (enable) {
            Log.i(tag, content);
        }
    }

    public static void e(String content) {
        if (enable) {
            Log.e(LogTag, content);
        }
    }

    public static void e(String tag, String content) {
        if (enable) {
            Log.e(tag, content);
        }
    }

}

File: utilsLibrary/src/main/java/com/zeropercenthappy/utilslibrary/LogUtils.kt
package com.zeropercenthappy.utilslibrary

import android.util.Log

object LogUtils {
    var LOG_TAG = "DEBUG"
    var ENABLE = false

    fun i(content: String) {
        if (ENABLE) {
            Log.i(LOG_TAG, content)
        }
    }

    fun v(content: String) {
        if (ENABLE) {
            Log.v(LOG_TAG, content)
        }
    }

    fun w(content: String) {
        if (ENABLE) {
            Log.w(LOG_TAG, content)
        }
    }

    fun e(content: String) {
        if (ENABLE) {
            Log.e(LOG_TAG, content)
        }
    }

    fun wtf(content: String) {
        if (ENABLE) {
            Log.wtf(LOG_TAG, content)
        }
    }
}